tests/test_macros.py

```diff
@@ -0,0 +1,26 @@
+import uuid
+
+import pytest
+from dbt.clients.jinja import MacroGenerator
+from pyspark.sql import SparkSession
+
+
+@pytest.fixture
+def simple_table(spark_session: SparkSession) -> str:
```

Comment:
@jtcohen6 : I moved this to a fixture so that the table is always deleted after a single test. This makes the test more clean and independent.

Reply:
I like it!